dynamodb/src/test/scala/docs/scaladsl/ExampleSpec.scala

```diff
@@ -31,83 +35,68 @@ class ExampleSpec
   implicit val materializer: Materializer = ActorMaterializer()
   override implicit val patienceConfig: PatienceConfig = PatienceConfig(5.seconds, 100.millis)
 
-  override def beforeAll() = {
-    System.setProperty("aws.accessKeyId", "someKeyId")
-    System.setProperty("aws.secretKey", "someSecretKey")
+  implicit val client: DynamoDbAsyncClient = DynamoDbAsyncClient
+    .builder()
+    .region(Region.AWS_GLOBAL)
+    .credentialsProvider(StaticCredentialsProvider.create(AwsBasicCredentials.create("x", "x")))
+    .endpointOverride(new URI("http://localhost:8001/"))
+    .build()
+
+  override def afterAll(): Unit = {
+    client.close()
+    shutdown()
+    super.afterAll()
   }
 
-  override def afterAll(): Unit = shutdown()
-
   "DynamoDB" should {
 
     "provide a simple usage example" in {
 
       //##simple-request
-      val listTablesResult: Future[ListTablesResult] =
-        DynamoDb.single(new ListTablesRequest())
+      val listTablesResult: Future[ListTablesResponse] =
+        DynamoDb.single(ListTablesRequest.builder().build())
       //##simple-request
 
       listTablesResult.futureValue
     }
 
     "allow multiple requests - explicit types" in assertAllStagesStopped {
```

Comment:
The ` - explicit types` is not relavant in the test name anymore, as that is not required in this improved API.